File: Client/Assets/iTweenExamples/TurretExample/Scripts/Crosshair.cs
using UnityEngine;
using System.Collections;

public class Crosshair : MonoBehaviour{
	void Update(){	
		Vector3 mousePos = new Vector3(Input.mousePosition.x,Input.mousePosition.y,Camera.main.transform.position.y);
		Vector3 worldPos = Camera.main.ScreenToWorldPoint(mousePos);
		iTween.MoveUpdate(gameObject,iTween.Hash("position",worldPos,"time",.6));
	}
}

File: Client/Assets/Package/Pixelplacement/iTweenExamples/TurretExample/Scripts/Crosshair.js
function Update(){	
	var mousePos : Vector3 = Vector3(Input.mousePosition.x,Input.mousePosition.y,Camera.main.transform.position.y);
	var worldPos : Vector3 = Camera.main.ScreenToWorldPoint(mousePos);
	iTween.MoveUpdate(gameObject,{"position":worldPos,"time":.6});
}